apps/nowcasting-app/constant.ts

```diff
@@ -1,4 +1,4 @@
-export const API_PREFIX = "https://api-dev.nowcasting.io/v0";
+export const API_PREFIX = process.env.NEXT_PUBLIC_API_PREFIX || "https://api-dev.nowcasting.io/v0";
```

Comment:
why the name 'NEXT_PUBLIC_API_PREFIX'? Could it be `OCF_API_PREFIX`

Reply:
@peterdudfield sorry just seen these comments - as the API stuff is currently read from the backend and frontend, you need to prefix any frontend-exposed vars with `NEXT_PUBLIC` to be accessible from the frontend. Can add OCF, I'm easy either way